static/styles/popovers.css

```diff
@@ -299,9 +299,10 @@ ul {
         height: 180px;
         width: 180px;
         background-size: cover;
-        background-position: center;
+        background-position: center center;
         border-radius: 5px;
         margin-right: 10px;
+        background-clip: content-box;
```

Comment:
Can you explain in the commit message why `center center` is correct?  It's the kind of thing that looks like a bug.

Check out the Zulip commit message guidelines for more details: https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-messages

Reply:
As we know,if we do not specify the other(second) parameter of background-position,it is taken as center by default.So its the same whether we write `center center` or just the `center` but to avoid it looking like like a bug or further confusions, i have removed the center center property value and made it the same as before that is only the center.I hope its correct now.